board/stm32h7/llfdcan.h

```diff
@@ -0,0 +1,232 @@
+#define FDCAN_MESSAGE_RAM_SIZE 0x2800UL
```

Comment:
why are these defines instead of typed constants?

Reply:
STM headers do have it like this